lib/temporal/workflow/poller.rb

```diff
@@ -60,6 +60,9 @@ def poll_for_task
         client.poll_workflow_task_queue(namespace: namespace, task_queue: task_queue)
       rescue StandardError => error
         Temporal.logger.error("Unable to poll workflow task queue: #{error.inspect}")
+
+        Temporal::ErrorHandler.handle(error, nil)
```

Comment:
I think this is expected to fail on a polling timeout, so I don't think it's super helpful to report. Maybe we need to rescue different errors here to distinguish expected failure

Reply:
If that were the case then we'd see "Unable to poll workflow task queue" all over the logs, but we don't. So I think failure here is unexpected.

Reply:
That's a good point, you're probably right. Please ignore it then